jekyll/_cci2/local-cli.md

```diff
@@ -22,26 +22,28 @@ You can also run `circleci` commands in your `config.yml` file for jobs that use
 
 ## Installing
 
-### Ubuntu 16.04+, Fedora 27+, and other Distros Supporting Snap Packages
+### macOS and Linux Distros
 
-With `snapd` installed (pre-installed on Ubuntu), you can install the Local CLI on various Linux distributions. The following commands give you the CircleCI Local CLI, Docker, and the security & auto-update features that come along with [Snap packages](https://www.ubuntu.com/desktop/snappy).
+1. Install and configure Docker by using the [docker installation instructions](https://docs.docker.com/engine/installation/).
+
+2. To install the CLI, run the following command:
 
 ```
-sudo snap install docker circleci
-sudo snap connect circleci:docker docker
+$ curl -o /usr/local/bin/circleci https://circle-downloads.s3.amazonaws.com/releases/build_agent_wrapper/circleci && chmod +x /usr/local/bin/circleci
 ```
 
-### macOS and other Linux Distros
+The CLI, `circleci`, is downloaded to the `/usr/local/bin` directory. If you do not have write permissions for `/usr/local/bin`, you might need to run the above commands with `sudo`. The CLI automatically checks for updates and will prompt you if one is available. 
 
-1. Install and configure Docker by using the [docker installation instructions](https://docs.docker.com/engine/installation/).
+### [Alternative] Ubuntu 16.04 & Ubuntu 17.04+
 
-2. To install the CLI, run the following command:
+With Ubuntu, you can optionally install via Snap. The following commands give you the CircleCI Local CLI, Docker, and the security & auto-update features that come along with [Snap packages](https://www.ubuntu.com/desktop/snappy).
 
 ```
-$ curl -o /usr/local/bin/circleci https://circle-downloads.s3.amazonaws.com/releases/build_agent_wrapper/circleci && chmod +x /usr/local/bin/circleci
+sudo snap install docker circleci
+sudo snap connect circleci:docker docker
 ```
 
-The CLI, `circleci`, is downloaded to the `/usr/local/bin` directory. If you do not have write permissions for `/usr/local/bin`, you might need to run the above commands with `sudo`. The CLI automatically checks for updates and will prompt you if one is available. 
+***note:*** With snap packages, the `docker` command will use the Docker snap, not any version of Docker you may have previously installed. For security purposes, snap packages can only read/write files from within `$HOME`.
```

Comment:
"note" should be capitalized. And are there extra asterisks around it? Did you mean to do that? If so, I think it's a bit of overkill for a word that's already bolded.

Reply:
Three means bold and italic.

Reply:
@smart-alek Is that an issue?

Reply:
I'd say bolding and italicizing undermines the effect of either one of those effects. Maybe @michelle-luna has some thoughts.